etna/transforms/detrend.py

```diff
@@ -53,13 +57,12 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
 
         Parameters
         ----------
-        df: pd.Series
-            data series to subtract trend from
+        df:
+            data to subtract trend from
 
         Returns
         -------
-            pd.Series
-                residue after trend subtraction
+            residue after trend subtraction
```

Comment:
could you please check that docs looks good with this indent in Return block? 
there is the guide how to build the docs https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md#documentation

Reply:
In this particular case it is not relevant because docs for all meaningful classes are inherited from `PerSegmentWrapper`.